Collection/postmanCollection.txt

```diff
@@ -0,0 +1,54 @@
+1. Channel Connector API:
+
```

Comment:
Exported postman collection file extension should be json or postman specific extension. Can you try loading this txt file into Postman and seeing if it works?

Reply:
resolved